lib/timer.py

```diff
@@ -12,20 +12,33 @@
 from uaclient.cli import setup_logging
 from uaclient.config import UAConfig
 from uaclient.jobs.update_messaging import update_apt_and_motd_messages
+from uaclient.jobs.update_state import update_status
 
 
 LOG = logging.getLogger(__name__)
+TIME_FREQUENCY_6_HOURS = 21600
+TIME_FREQUENCY_12_HOURS = TIME_FREQUENCY_6_HOURS * 2
```

Comment:
these are going to go away then and be replaced by reading config values with a default set in case they're not set then?

Reply:
Yes, those values are the default ones. If the user intends to add a custom configuration on `uaclient.conf` they will override those values

Reply:
Right, but then why wouldn't it be job focused? Right now if you were to change the time you need a whole new variable name. 

I'd suggest just  

    UPDATE_MESSAGING_INTERVAL = xxx
    UPDATE_STATUS_INTERVAL = yyy

and then you can use those in the code and if they ever change it's all good who cares. 

Once the config is there it could be something like

    config.get('update_messaging_interval', UPDATE_MESSAGING_INTERVAL) 

and you're off to the races.

Reply:
Fair enough, that's a good point. I will update the code